doc/sphinx/source/esmvalcore/recipe.inc

```diff
@@ -0,0 +1,5 @@
+.. _recipe:
```

Comment:
It seems like the above files start with double colon (::) instead of double dot (..). I am not fluent enough in rst to know if this is what you want or not.